libutil/dr_config.c

```diff
@@ -682,7 +692,17 @@ write_config_param(FILE *f, const char *var, const TCHAR *val)
     int len;
     char buf[MAX_CONFIG_VALUE];
     DO_ASSERT(f != NULL);
+#    ifdef __GNUC__
+#        ifndef __clang__
+#            pragma GCC diagnostic ignored "-Wformat-truncation"
```

Comment:
Could we share and shrink this to avoid 5 identical lines in multiple places?  With _Pragma we could have a single line `DISABLE_FORMAT_TRUNCATION_START` and then `DISABLE_FORMAT_TRUNCATION_STOP` (or parameterized by the -W).

Or just put a `XXX` comment about doing so in the future if more of these than these 2 are added.